README.md

```diff
@@ -73,39 +78,27 @@ pip install -r requirements/gui.txt
 After this, the command `python joinmarket-qt.py` from within the `scripts` subdirectory should work.
 There is a [walkthrough](docs/JOINMARKET-QT-GUIDE.md) for what to do next.
 
-### Notes on architectural changes (can be ignored)
+### Architecture notes
 
-Motivation: By separating the code which manages conversation with other
-Joinmarket participants from the code which manages this participant's Bitcoin
-wallet actions, we get a considerable gain at a minor cost of an additional layer:
-code dependencies for each part are much reduced, security requirements of the 
-server/daemon layer are massively reduced (which can have several advantages such as
-it being more acceptable to distribute this layer as a binary), and client code
-can be written, implementing application-level logic (do join with coins X under condition X)
-using other Bitcoin libraries, or wallets, without knowing anything about
-Joinmarket's inter-participant protocol. An example is my work on the Joinmarket
-electrum [plugin](https://github.com/AdamISZ/electrum-joinmarket-plugin).
+See [architecture-notes.md](docs/architecture-notes.md).
 
-It also
-means that updates to the Bitcoin element of Joinmarket, such as P2SH and segwit, should
-have extremely minimal to no impact on the backend code, since the latter just implements
-communication of a set of formatted messages, and allows the client to decide on
-their validity beyond simply syntax.
+### TESTING
 
-Joinmarket's own [messaging protocol](https://github.com/JoinMarket-Org/JoinMarket-Docs/blob/master/Joinmarket-messaging-protocol.md) is thus enforced *only* in the server/daemon.
+Instructions for developers for testing [here](docs/TESTING.md). If you want to help improve the project, please have a read of [this todo list](docs/TODO.md).
 
-The client and server currently communicate using twisted.protocol.amp, see
-[AMP](https://amp-protocol.net/),
-and the specification of the communication between the client and server is isolated to
-[this](https://github.com/AdamISZ/joinmarket-clientserver/blob/master/jmbase/jmbase/commands.py) module.
-Currently the messaging layer of Joinmarket is IRC-only (but easily extensible, see [here](https://github.com/JoinMarket-Org/joinmarket/issues/650).
-The IRC layer is also implemented here using Twisted, reducing the complexity required with threading.
+### Community
 
-The "server" is just a daemon service that can be run as a separate process (see `scripts/joinmarketd.py`), or for convenience in the same process (the default for command line scripts).
++ IRC: `#joinmarket` on irc.freenode.net https://webchat.freenode.net/?channels=%23joinmarket
 
-### TESTING
++ IRC on tor: `#joinmarket` on the networks [AgoraIRC](https://anarplex.net/agorairc/) and [Cyberguerilla](https://cyberguerrilla.info/). These channels are bridged to the above freenode channel.
```

Comment:
Freenode also supports Tor, maybe worth mentioning? https://freenode.net/news/tor-online

Reply:
TIL. Last time I checked that wasn't possible. But there is also this problem: `It's currently not possible to register an account for use with Tor without connecting at least once over the Internet.`. Connecting over the clearnet once doxes you to freenode.

Reply:
Ok, yes, maybe not worth mentioning.